### ISSUE TITLE
Cannot open project after QR Code scan

### ISSUE THREAD

**Issue by russbiggs** [sharedstreets/curb-wheel-app#24]
seems like theres a memory leak that prevents opening a project right after scanning.  Log output shows the following:

```
E/flutter (10114): #2      _QrScannerState._onQRViewCreated.<anonymous closure>
package:curbwheel/…/projects/qr_scanner.dart:95
E/flutter (10114): #3      _rootRunUnary (dart:async/zone.dart:1198:47)
E/flutter (10114): #4      _CustomZone.runUnary (dart:async/zone.dart:1100:19)
E/flutter (10114): #5      _CustomZone.runUnaryGuarded (dart:async/zone.dart:1005:7)
E/flutter (10114): #6      _BufferingStreamSubscription._sendData (dart:async/stream_impl.dart:357:11)
E/flutter (10114): #7      _DelayedData.perform (dart:async/stream_impl.dart:611:14)
E/flutter (10114): #8      _StreamImplEvents.handleNext (dart:async/stream_impl.dart:730:11)
E/flutter (10114): #9      _PendingEvents.schedule.<anonymous closure> (dart:async/stream_impl.dart:687:7)
E/flutter (10114): #10     _rootRun (dart:async/zone.dart:1182:47)
E/flutter (10114): #11     _CustomZone.run (dart:async/zone.dart:1093:19)
E/flutter (10114): #12     _CustomZone.runGuarded (dart:async/zone.dart:997:7)
E/flutter (10114): #13     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1037:23)
E/flutter (10114): #14     _rootRun (dart:async/zone.dart:1190:13)
E/flutter (10114): #15     _CustomZone.run (dart:async/zone.dart:1093:19)
E/flutter (10114): #16     _CustomZone.runGuarded (dart:async/zone.dart:997:7)
E/flutter (10114): #17     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1037:23)
E/flutter (10114): #18     _microtaskLoop (dart:async/schedule_microtask.dart:41:21)
E/flutter (10114): #19     _startMicrotaskLoop (dart:async/schedule_microtask.dart:50:5)
```

**Comment by kpwebb** [comment]
need confirm if this is fixed/still issue with recent changes @russbiggs 

Also @kpwebb needs to check docs to see if we're cleaning up camera on page dispose.

**Comment by kpwebb** [comment]
couldn't replicate but fixed potential source of problem in #68

**Comment by kpwebb** [comment]
Looks good!
